src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/custom.py

```diff
@@ -0,0 +1,390 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+
+import json
+import os
+import signal
+import time
+import requests
+from azure.cli.core.keys import is_valid_ssh_rsa_public_key
+
+from azure.cli.core.util import CLIError
+import azure.mgmt.batchai.models as models
+
+from msrest.serialization import Deserializer
+
+# Environment variables for specifying azure storage account and key. We want the user to make explicit
+# decision about which storage account to use instead of using his default account specified via AZURE_STORAGE_ACCOUNT
+# and AZURE_STORAGE_KEY.
+AZURE_BATCHAI_STORAGE_ACCOUNT = 'AZURE_BATCHAI_STORAGE_ACCOUNT'
+AZURE_BATCHAI_STORAGE_KEY = 'AZURE_BATCHAI_STORAGE_KEY'
```

Comment:
Should use the config system in the CLI instead of using env vars directly.
The config system works like this - https://github.com/Microsoft/knack/blob/master/docs/config.md.

E.g. we use this for the default output type in the CLI - `az_config.get('core', 'output', fallback='json')` which allows someone to set the `AZURE_CORE_OUTPUT` env var or can set it in the config file also.

https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/application.py#L272

This is also another good example - https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py#L79.

Reply:
hm, we would like to behave the same way as storage. we already asking user to setup AZURE_STORAGE_ACCOUNT and AZURE_BATCHAI_STORAGE account and it's not very friendly. Asking user to provide them via different mechanisms will be even more confusing... what do you think?